extensions/dbal/lib/Storage/Driver/Dbal/Schema.php

```diff
@@ -40,7 +40,7 @@ private function createRun()
         $table = $this->createTable('run');
         $table->addColumn('id', 'integer', ['autoincrement' => true]);
         $table->addColumn('uuid', 'string');
-        $table->addColumn('context', 'string', ['notnull' => false]);
+        $table->addColumn('tag', 'string', ['notnull' => false]);
```

Comment:
probably going to remove the dbal storage at some point, worth having the BC break? (only needs a schema update ..)